pennylane/devices/__init__.py

```diff
@@ -151,12 +151,13 @@ def execute(self, circuits, execution_config = qml.devices.DefaultExecutionConfi
 
 # DefaultQubitTF and DefaultQubitAutograd not imported here since this
 # would lead to an automatic import of tensorflow and autograd, which are
-# not PennyLane core dependencies
+# not PennyLane core dependencies.
+# DefaultTensor is not imported here to avoid warnings
+# from quimb in case it is installed on the system.
 from .default_qubit_legacy import DefaultQubitLegacy
 from .default_gaussian import DefaultGaussian
 from .default_mixed import DefaultMixed
 from .default_clifford import DefaultClifford
 from .null_qubit import NullQubit
 from .default_qutrit_mixed import DefaultQutritMixed
-from .default_tensor import DefaultTensor
```

Comment:
So the difference here is that `qml.devices.DefaultTensor` is no longer accessible but has to be imported explicitly as `from pennylane.devices.default_tensor import DefaultTensor`, right?
Do we mind? If we can instantiate the device with `qml.device` it's okay. But it seems that is not supported yet, so maybe this explicit import requirement can be a bit of an annoyance? I personally think that the warnings are the bigger annoyance.

Reply:
Thanks @dwierichs!

>  If we can instantiate the device with qml.device it's okay. But it seems that is not supported yet

That should be supported in every system. Do you encounter errors with:

```
qml.device("default.tensor", wires=2)
```

?

Reply:
Yes, it tells me that the plugin is not installed :sweat_smile: I am on master and have quimb installed. :thinking:

Reply:
That's strange. The `default.tensor` entry point in the `setup.py` file is [currently in master](https://github.com/PennyLaneAI/pennylane/blob/master/setup.py#L63). On my system, it works just fine. Is there's something I'm missing?

Reply:
Sorry, must have been a hickup, it works after reinstalling pennylane :shrug: